### PR TITLE
[composer]Attached files show up as attachments in draft emails and when replying to emails

### PR DESCRIPTION
**Pull request by heathergh**
# Code changes

- Attachments now appear in reply and draft messages

# Readiness checklist

- [ ] Added changes to component `CHANGELOG.md`
- [ ] New property added? make sure to update `component/src/properties.json`
- [ ] Cypress tests passing?
- [ ] New cypress tests added?
- [ ] Included before/after screenshots, if the change is visual

# License

I confirm that this contribution is made under the terms of the MIT license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Review by krisgardiner — APPROVED** [review]
Looks good, can we update the changelog please

**Review by pooja169usp — COMMENTED** [review]
@heathergh the attachments should also show up when the message is forwarded. Currently it does not work for forwarding.

File: components/composer/src/Composer.svelte [hunk 1]
```diff
@@ -70,6 +71,7 @@
   export let access_token: string = "";
 
   export let value: Message | void;
+  export let message_body: Message | void;
```
Comment (pooja169usp) [inline components/composer/src/Composer.svelte:74]: I think we could rename `message_body` to `message` or `message_object`. As `message_body` seems like it refers to Message.body. Wdyt @heathergh ?

Comment (pooja169usp) [reply components/composer/src/Composer.svelte:74]: Also, we have a prop called `template` which technically is the message body.

**Review by pooja169usp — APPROVED** [review]
Thanks for fixing the attachment issue @heathergh!
Everything else looks good, just 1 comment!

File: components/composer/src/Composer.svelte (hunk 1 above)
Comment (heathergh) [reply components/composer/src/Composer.svelte]: Agreed! I changed it to `message_with_body`

**Pull request merged**